pyproject.toml

```diff
@@ -31,7 +31,8 @@ dependencies = [
     "marshmallow_oneofschema>=2.0.1",
     "backoff>=2.0.0",
     "pydantic>=1.10.0",
-    "PyJWT>=2.8.0"
+    "PyJWT>=2.8.0",
+    "ansys-hps-data-transfer-client@git+https://github.com/ansys-internal/hps-data-transfer-client.git@main#egg=ansys-hps-data-transfer-client"
```

Comment:
@FedericoNegri @RajaPiseD I want to bring to your attention that adding dependencies using the git specifier is not valid when uploading packages to PyPI. The Python Package Index will reject any uploads containing dependencies specified like this.

Reply:
@jorgepiloto we're aware, that's currently an internal dependency which is meant to either be incorporated in pyhps directly or become an open source pyansys package on its own. Either way, it's a prerequisite for the next pyhps release.